### ISSUE TITLE
Provide data about requesting responsible

### ISSUE THREAD

**Issue by mikaelrss** [Altinn/digital-gravferd#97]
### Description

Provide an endpoint in dgm-gravferden which provides selections about the funeral made.

**Comment by chrrust** [comment]
We will need to either derive the name of the gravferdsbyrå from the organization number of the gravferdsbyrå, or to pass it as part of the form data when the user submits the ansvarsmeldingskjema. I'm tempted to say we add it amoung the submitted data to keep the list of gravferdsbyråer in one place. Thoughts, @mikaelrss ?

**Comment by mikaelrss** [comment]
That could present problems if the funeral agencies updates their name after a user submits the form. I think it would be best if we did a lookup on org number. We want to make registering as a funeral agency a service anyways, so that service could maybe also provide an API for looking up the name?

**Comment by chrrust** [comment]
Good catch! Yes, that's right. Alright, so what if we add a new issue where we move the list of funeral agencies to `dgm-gravferden` and create an endpoint for retrieving those?

**Comment by mikaelrss** [comment]
I think that sounds good. We might want to extract it to a separate service in the future

**Comment by chrrust** [comment]
#99 Created this issue and added it to the current sprint due to this issue already being in the sprint. Please review it

**Comment by mikaelrss** [comment]
My thoughts: 
- I think I would rename `RolleTilDenDøde` to `RelasjonTilDenDøde`.
- And let's not return `GravferdsbyråNavn` but instead let consumers resolve this. 
- We create a top level object `ForespurtAnsvarlig` which includes an `Ansvarsmelding` and use this as the response: 
```c#
public class ForespurtAnsvarlig
{
  public Ansvarsmelding Ansvarsmelding { get; set; }
  public RelasjonTilDenDøde RelasjonTilDenDøde // This is the calling partys relation to the dead
}
```

**Comment by mikaelrss** [comment]
And it seems like we're missing a field for `HarInformertAndreEtterlatte`

**Comment by mikaelrss** [comment]
And we would also need an option for `AskespredningsValg`

**Comment by mikaelrss** [comment]
Sounds like a good solution!

**Comment by chrrust** [comment]
Seems to not be supported as per now. Apps have an issue on it, but it was created a long while ago.
https://github.com/Altinn/app-frontend-react/issues/265

**Comment by mikaelrss** [comment]
We can have a look into it and see if we can contribute anything to the apps-team on this front?

**Comment by chrrust** [comment]
Absolutely. I recon it'd be a useful features for others aswell

**Comment by chrrust** [comment]
Blocked due to the need of some clarifications regarding the domain technical aspects of changing the current responsible person. I'm pitching an agenda point to the thursdag meeting to further discuss this matter.
